src/store/reducers/formsReducer.ts

```diff
@@ -0,0 +1,29 @@
+import { ActionType } from '../../types/store/actionTypes';
+import { formsActionTypes } from '../actionTypes/formsActionTypes';
+import { FormInitState } from '../../types/store/initStateInterfaces';
+
+const initState: FormInitState = {
+  name: '',
+  description: ''
+};
+
+const formsReducer = (state: FormInitState = initState, action: ActionType) => {
```

Comment:
it is better to rely on useReducer from react to handle such logic. (you will not mutate a global state in a such scenario)
But better to use formik or similar form-control manager